main.go

```diff
@@ -190,7 +190,8 @@ func handleGoogleCallback(s sessions.Store) http.Handler {
 
 		session.Values["email"] = gp.Email
 		session.Values["GoogleID"] = gp.ID
-		session.Values["valid_until"] = time.Now().UTC().Add(10080 * time.Minute)
+		SessionValidTimeInMin := time.Duration(config.SessionValidTime)
```

Comment:
NIT: Should we name this variable more accurately? I believe `time.Duration()` uses `ns` as units. so maybe `SessionValidTimeInNS`